Add docs about debugging (#37)
saleor/saleor-app-sdk: docs/debugging.md

# Debugging

> Ability to debug is important for apps developers. App SDK provides opt-in verbose logs that can be used

Reviewer: ```suggestion
The ability to debug is important for app developers. App SDK provides opt-in verbose logs that can be used.
```

Also, the sentence:
> App SDK provides opt-in verbose logs that can be used

reads like it was cut short. Is there supposed to be something after "can be used"? Maybe it needs to be rephrased.

Author: Rephrased + applied all suggestions

> Debug logs are implemented with [debug](https://www.npmjs.com/package/debug) package and work both in Node and Browser

Reviewer: ```suggestion
Debug logs are implemented with [debug](https://www.npmjs.com/package/debug) package and work both in Node and Browser.
```

## Enabling verbose logs

> Debug logs are aggregated within [namespaces](#namespaces). They can be all enabled, disabled, or confugred.

Reviewer: ```suggestion
Debug logs are aggregated within [namespaces](#namespaces). They can be enabled, disabled, or configured.
```

> By default, all verbose logs are disabled

Reviewer: ```suggestion
By default, all verbose logs are disabled.
```

> To enable logs, use following:

Reviewer: ```suggestion
To enable logs, use the following:
```

### Enabling in Node context

> Add `DEBUG={namespaces}` env variable

Reviewer: ```suggestion
Add `DEBUG={namespaces}` env variable.
```

#### Example

```shell
DEBUG=* pnpm run dev
```

> For more details check [debug package](https://github.com/debug-js/debug#usage)

Reviewer: ```suggestion
For more details check [debug package](https://github.com/debug-js/debug#usage).
```

### Enabling in Browser context

> Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support)

Reviewer: ```suggestion
Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support).
```

#### Example

```javascript
// In devtools console
localStorage.debug = "*";
```

## Namespaces

> Use namespace name to enable debug logs for each module

Reviewer: ```suggestion
Use the namespace name to enable debug logs for each module.
```

# Debugging

The ability to debug is important for app developers.
App SDK provides opt-in verbose logs, that can be enabled for specific modules.

Debug logs are implemented with [debug](https://www.npmjs.com/package/debug) package and work both in Node and Browser.

## Enabling verbose logs

Debug logs are aggregated within [namespaces](#namespaces). They can be enabled, disabled, or configured.

By default, all verbose logs are disabled.

To enable logs, use the following:

### Enabling in Node context

Add `DEBUG={namespaces}` env variable.

#### Example

```shell
DEBUG=* pnpm run dev
```

For more details check [debug package](https://github.com/debug-js/debug#usage).

### Enabling in Browser context

Set `localStorage.debug = '{namepsaces}'` in your frontend code. For more details see [debug package](https://github.com/debug-js/debug#browser-support).

#### Example

```javascript
// In devtools console

> localStorage.debug = "*";

Author: Good one, will add
